pom.xml

```diff
@@ -67,7 +68,7 @@
    </dependency>
      <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-client</artifactId>
```

Comment:
I can't remember why I replaced hadoop-client with hadoop-common.  I may have thought there was a class missing that we needed from the common jar.  I just built and tested it again and hadoop-client seems to work just fine.